css/media-queries.md

```diff
@@ -57,113 +57,114 @@ Everyone.  Some companies with very responsive sites are [Etsy](https://www.etsy
-    </p>
-    <img src="humpback.jpg" alt="whale">
-    <img src="yellow-fish.jpg" alt="yellow fish">
-    <img src="red-fish.jpg" alt="red fish">
-  </body>
-</html>
-```
+	```html
+	<!DOCTYPE html>
+	<html lang="en" dir="ltr">
+		<head>
+			<meta charset="utf-8">
+			<link rel="stylesheet" href="index.css">
+			<title></title>
+		</head>
+		<body>
+			<h1>Under the Sea</h1>
+			<p>
+				Halfbeak Blind shark, Australian herring filefish elver bent-tooth Russian sturgeon koi, mud cat; Celebes rainbowfish tilapia swordtail. Tang flagtail, pompano dolphinfish jewel tetra thornyhead; mako shark. Wallago hussar, longnose whiptail catfish, "scup, yellow perch cutthroat trout Blind shark driftwood catfish Atlantic cod Blenny river loach?" Splitfin rocket danio barbeled houndshark velvetfish sand tiger golden shiner cowfish snake mackerel, "baikal oilfish porgy fusilier fish creek chub mud catfish." Coelacanth bent-tooth spearfish: soapfish mahseer Russian sturgeon fire bar danio kelp perch. Blind goby shortnose greeneye Celebes rainbowfish bigmouth buffalo: yellowtail, earthworm eel ghost knifefish amur pike. Gray mullet aholehole steelhead Australasian salmon barfish Siamese fighting fish shortnose chimaera inanga yellow bass opaleye.
+				Leopard danio pencilsmelt thorny catfish razorfish boarfish barreleye, sand tiger. Walking catfish tailor--cuchia prickly shark chain pickerel Port Jackson shark sawtooth eel turkeyfish slimehead South American Lungfish torrent catfish. Ayu bull trout trumpeter; hussar, buri shad, lanternfish coolie loach, "mud cat: crappie pike eel; man-of-war fish." Shiner bamboo shark Black angelfish Rabbitfish Russian sturgeon coolie loach sablefish king-of-the-salmon. Brook lamprey Pacific herring prickly shark shiner sawtooth eel eel cod medaka. Redfish bala shark flounder, Old World knifefish Black prickleback large-eye bream. Long-whiskered catfish Manta Ray paradise fish deep sea smelt vendace skilfish sea raven brotula. Hagfish Long-finned sand diver tilefish knifefish Ganges shark New Zealand smelt."
+			</p>
+			<img src="humpback.jpg" alt="whale">
+			<img src="yellow-fish.jpg" alt="yellow fish">
+			<img src="red-fish.jpg" alt="red fish">
+		</body>
+	</html>
+	```
 1. Create a CSS file, and style your images:
-```
-img {
-	width: 32%
-}
-```
+	```css
+	img {
+		width: 32%
```

Comment:
This spacing is just to much... it’s 8, I think, when 2 will do. Do you see what I mean?
```suggestion
        width: 32%
```